app/src/org/commcare/activities/components/ImageCaptureProcessing.java

```diff
@@ -141,17 +144,21 @@ private static void processImageGivenFileUri(FormEntryActivity activity, String
             return;
         }
 
-        // First make a copy of the image to operate on and then pass it to the File function
-        File finalFile = new File(CommCareApplication.instance().
-                getExternalTempPath(FileUtil.getFileName(activity, imageUri)));
         try {
+            // First make a copy of the image to operate on and then pass it to the File function
+            File finalFile = new File(CommCareApplication.instance().
+                    getExternalTempPath(FileUtil.getFileName(activity, imageUri)));
             FileUtil.copyFile(inputStream, finalFile);
-        } catch (IOException e) {
+            processImageGivenFilePath(activity, instanceFolder, finalFile.getAbsolutePath());
+        } catch (FileExtensionNotFoundException e) {
```

Comment:
These two blocks are actually quite similar to each other, I think it would be fine to follow the same catch (IE: Logger.exception+toast+return) for both cases.